NAMESPACE

```diff
@@ -166,6 +166,10 @@ importFrom(htmltools,tagList)
 importFrom(htmltools,tags)
 importFrom(lifecycle,deprecated)
 importFrom(stats,na.omit)
+importFrom(stringi,stri_detect)
+importFrom(stringi,stri_replace)
+importFrom(stringi,stri_sub)
+importFrom(stringi,stri_wrap)
```

Comment:
@Melkiades @pawelru , what do you think of these? I was wondering if we should keep using "::" while we had things with importFrom

Reply:
i think this PR is almost good to go apart from "::" issue. what do you guys think?

Reply:
The importFrom is to add entries to the NAMESPACE via namespace roclet during roxygenize process. You must have at least one entry in the NAMESPACE as otherwise R CMD CHECK with throw a warning that you have package in Imports that is not you are not using in the NAMESPACE. You can have multiple entries. For me it's a little bit more accurate so I would recommend to do this but the absolute minimum is to have at least one.

Prefixing with `::` is a little bit separate. This is a safeguard for namespace collision in case multiple imported packages export objects of identical names. I'm personally a big fan of robust code and I apply prefixes everywhere. 

Without the prefix, if you import any function from the package via NAMESPACE then that package would be attached and (assuming no collision) a function can be successfully found and used. Prefixing is not obligatory but I just consider these as a good practice of writing the code. 

Now I don't want to ask you to prefix everything because this is a big heavylift. It's just not to remove what's already there. It's somewhat a step back for code quality